dataflow-guides/cloud-backup.md

```diff
@@ -1,45 +1,60 @@
-Description of guide
+This guide will walk you through how to configure and test running a dataflow with cloud recovery. To do so we will start with an AWS kubernetes cluster and then migrate the same dataflow to a Gooogle Cloud Kubernetes cluster. Having an account a cluster in each is a prerequisite for trying this.
+
+Cloud recovery is a capability that allows the state persisted in the local worker to be written to object storage like s3. In the event of a node dying, you will be able to recover dataflows from this bucket. This can also aid in Kubernetes cluster migration without the loss of state.
```

Comment:
Recovering from an event of a node dying is also covered with the standard recovery feature. I mean, without having cloud backup enabled.

We could mention "a cluster disaster or the whole cluster went down."